pkg/controller/prpqr_reconciler/testdata/zz_fixture_prpqr_TestReconcile_basic_case__prowjob_already_exists__no_updates.yaml

```diff
@@ -34,3 +34,9 @@
       reason: AllJobsTriggered
       status: "True"
       type: AllJobsTriggered
+    jobs:
+    - jobName: periodic-ci-test-org-test-repo-test-branch-test-name
+      prowJob: some-uuid
+      status:
+        startTime: "1970-01-01T00:00:00Z"
+        state: triggered
```

Comment:
This change shows one of the impacts of the change - the job status was reconciled because it was previously missing in the resource.